jest.config.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * @returns {Promise<import('jest').Config>}
+ */
+const path = require('path');
+
+module.exports = {
+  collectCoverageFrom: [
+    '**/src/**/*.component.tsx',
+    '!**/node_modules/**',
+    '!**/vendor/**',
+    '!**/src/**/*.test.*',
+    '!**/src/declarations.d.ts',
+    '!**/e2e/**',
+  ],
+  transform: {
+    '^.+\\.tsx?$': ['@swc/jest'],
+  },
+  transformIgnorePatterns: ['/node_modules/(?!@openmrs)'],
+  moduleNameMapper: {
+    '@openmrs/esm-framework': '@openmrs/esm-framework/mock',
+    '@openmrs/esm-utils': '@openmrs/esm-framework/mock',
+    '\\.(s?css)$': 'identity-obj-proxy',
+    '^lodash-es/(.*)$': 'lodash/$1',
```

Comment:
```suggestion
    '^lodash-es/(.*)$': 'lodash/$1',
    '^lodash-es$': 'lodash',
```

You'll need this entry too.